tools/create-translation-issues.sh

```diff
@@ -0,0 +1,216 @@
+
+declare -A TRANSLATORS_BY_LANG=(
+    # Syntax:
+    # [TYPE_LANG]="github-handle1,github-handle2"
+    # with TYPE being either app or web and
+    # with LANG being the language code (different syntax for app and web!)
+
+    # App translators:
+    [app_de_DE]="rolamos"
+    [app_es_ES]="ignotus666"
+    [app_fr_FR]="jujudusud"
+    [app_it_IT]="dzpex"
+    [app_nl_NL]="henkdegroot,jerogee"
+    [app_pl_PL]="SeeLook"
+    [app_pt_BR]="melcon"
+    [app_pt_PT]="Snayler"
+    [app_sk_SK]="jose1711"
+    [app_sv_SE]="genesisproject2020"
+
+    # Web translators:
+    [web_de]="Helondeth,ewarning"
+    [web_es]="ignotus666"
+    [web_fr]="jujudusud,trebmuh"
+    [web_it]="dzpex"
+)
+
+BODY_TEMPLATE_APP='Hi ${SPLIT_TRANSLATORS},
+We are getting ready for the ${RELEASE} release. No further changes to translatable strings are currently planned for this release.
+
+We would be happy if you updated the Jamulus software translations for **${LANG}** until **${DEADLINE}**.
+
+Please
+- Update your fork to current `master`,
+- Update translations using Qt Linguist in your fork,
+- Commit and push your changes and reference this Issue,
+- Open a Pull Request before ${DEADLINE}
+
+Suggested commit and PR message:
+```
+${TITLE}
+
+Fixes #<Insert this issue'"'"'s number here>
+```
+
+${EXTRA_TEXT}${MULTIPLE_TRANSLATORS_TEXT}
+
+Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).
+
+Thanks for contributing to Jamulus!'
+
+
+BODY_TEMPLATE_WEB='Hi ${SPLIT_TRANSLATORS},
+
+We are getting ready for the ${RELEASE} release and have created the [${TRANSLATE_BRANCH}](https://github.com/jamulussoftware/jamuluswebsite/tree/${TRANSLATE_BRANCH}) branch ([full diff](https://github.com/jamulussoftware/jamuluswebsite/compare/release..${TRANSLATE_BRANCH})).
+
+Because the English is still going through large changes, we are not expecting translations of anything other than the following pages:
+
+- [Home page](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/1-index.html)
+- [Setup](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Setup.md) (renamed from Getting-Started)
+- Installation pages for each supported platform ([Windows](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Installation-for-Windows.md), [Mac](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Installation-for-Macintosh.md), [Linux](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Installation-for-Linux.md))
+
+While you are of course welcome to translate more than this, please be aware that you may have to re-translate for the next release.
+
+We would be happy if you updated the translations for **${LANG}** until **${DEADLINE}**.
+
+Please
+
+- Start your work in your fork on a branch based on jamuluswebsite'"'"'s `${TRANSLATE_BRANCH}` branch.
```

Comment:
`- Start your work in your fork on a branch based on jamuluswebsite'"'"'s ``${TRANSLATE_BRANCH}`` branch.`

The code version is already clearer...  This needs to make it clear that local fork's master needs updating from jamulussoftware/jamuluswebsite master, then the local ``${TRANSLATE_BRANCH}`` branch updated, and then work started on a branch from the ``${TRANSLATE_BRANCH}`` branch.  If I've understood the process right...

Reply:
There is no `master` in the website repo (`release` would be the most similar branch). However, `${TRANSLATE_BRANCH}` is the only branch which translators should have to care about (well, 3.8.0 was different for $REASONS...). This branch is a one-time branch (e.g. `translate3_8_0`), so there should be no need to update/rebase.
As the process is supposed to be reworked regarding tooling (po4a) and branching, I'd like to keep it like this for now.

(cc @ignotus666 @ann0see @gilgongo)

Reply:
OK, if it's all going to change, no problem.  (However, make whatever's written clear enough that a first time translator will be able to read it and know what steps to follow.)